### PR TITLE
CDD-2598: Added version 2

### PR DESCRIPTION
**Pull request by rashadmughal**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -183,5 +183,21 @@ microservice {
...
+        stub {
+          host = localhost
+          port = 9478
+          bearer-token = "real"
+          context = /inventorylinking/exportsinbound/v2
+        }
+      }
+    }
   }
 }
```
Comment (paulblakehmrc) [inline conf/application.conf]: new line at end of file

File: public/api/conf/2.0/application.raml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+      displayName: Inventory Exports Request
+      description: Submit an Inventory Exports Request
+      headers:
+        Accept:
+          description: Specifies the version of the API that you want to call. See [versioning](/api-documentation/docs/reference-guide#versioning).
+          type: string
+          required: true
+          example: application/vnd.hmrc.1.0+xml
+        Content-Type:
+          description: Specifies the format of the request body, which must be XML. Download [schemas](/api-documentation/docs/api/download/customs-inventory-linking-exports/1.0/inventory-linking-exports-schemas.zip).
```
Comment (paulblakehmrc) [inline public/api/conf/2.0/application.raml]: url should contain `2.0`, not `1.0`

File: public/api/conf/2.0/application.raml [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+  /:
+    post:
+      displayName: Inventory Exports Request
+      description: Submit an Inventory Exports Request
+      headers:
+        Accept:
+          description: Specifies the version of the API that you want to call. See [versioning](/api-documentation/docs/reference-guide#versioning).
+          type: string
+          required: true
+          example: application/vnd.hmrc.1.0+xml
```
Comment (paulblakehmrc) [inline public/api/conf/2.0/application.raml]: example should read `application/vnd.hmrc.2.0+xml`, not `application/vnd.hmrc.1.0+xml` (as this is the core of how versioning works)

File: conf/application.conf (hunk 1 above)
Comment (glidester) [reply conf/application.conf]: Done

File: public/api/conf/2.0/application.raml (hunk 3 above)
Comment (glidester) [reply public/api/conf/2.0/application.raml]: Fixed

File: public/api/conf/2.0/application.raml (hunk 2 above)
Comment (glidester) [reply public/api/conf/2.0/application.raml]: Fixed

**Pull request merged**